### ISSUE TITLE
Add support for Gradle 6.0

### ISSUE THREAD

**Issue by wolfs** [gradle/gradle-profiler#170]
There should be some test coverage for Gradle 6.0. At least the `build-scan` profiler needs to be fixed.

**Comment by lptr** [comment]
cc @ldaley

**Comment by eskatos** [comment]
https://github.com/gradle/gradle-profiler/pull/173 adds Gradle 6.0 to the coverage matrix. It also limit the `buildScan` profiler tests to the latest 5.x as it doesn't work anymore with 6.0, what this very issue is about.

**Comment by guilherme-pereira-ifood** [comment]
I got this error when running with Gradle 6.0.1:

```
Caused by: org.gradle.internal.scan.config.UnsupportedBuildScanPluginVersionException: The build scan plugin is not compatible with this version of Gradle.
Please see https://gradle.com/help/gradle-6-build-scan-plugin for more information.
	at org.gradle.internal.scan.config.BuildScanConfigManager.collect(BuildScanConfigManager.java:111)
	at com.gradle.scan.plugin.internal.b.a$a.b(SourceFile:32)
	at com.gradle.scan.plugin.internal.b.a$a.a(SourceFile:28)
	at com.gradle.scan.plugin.internal.b.a.a(SourceFile:22)
	at com.gradle.scan.plugin.BuildScanPlugin.apply(SourceFile:179)
	at com.gradle.scan.plugin.BuildScanPlugin.apply(SourceFile:125)
	at org.gradle.api.internal.plugins.ImperativeOnlyPluginTarget.applyImperative(ImperativeOnlyPluginTarget.java:43)
	at org.gradle.api.internal.plugins.RuleBasedPluginTarget.applyImperative(RuleBasedPluginTarget.java:51)
	at org.gradle.api.internal.plugins.DefaultPluginManager.addPlugin(DefaultPluginManager.java:181)
	at org.gradle.api.internal.plugins.DefaultPluginManager.access$300(DefaultPluginManager.java:51)
	at org.gradle.api.internal.plugins.DefaultPluginManager$AddPluginBuildOperation.run(DefaultPluginManager.java:276)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:402)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:394)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor$1.execute(DefaultBuildOperationExecutor.java:165)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)
	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:92)
	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
	at org.gradle.api.internal.plugins.DefaultPluginManager$2.execute(DefaultPluginManager.java:159)
	at org.gradle.api.internal.plugins.DefaultPluginManager$2.execute(DefaultPluginManager.java:156)
	at org.gradle.configuration.internal.DefaultUserCodeApplicationContext.apply(DefaultUserCodeApplicationContext.java:49)
	at org.gradle.api.internal.plugins.DefaultPluginManager.doApply(DefaultPluginManager.java:156)
	... 169 more

* Results written to /Users/guilherme.pereira/Documents/Projects/ifood-consumer-android/profile-out-4
  Scenario using Gradle 6.0.1
```

**Comment by wolfs** [comment]
The build scan plugin profiler should now work again.

@guilherme-pereira-ifood: Can you try latest `master`?

**Comment by wolfs** [comment]
Fixed by #190.
